### ISSUE TITLE
Suggestion: Change the top right "+" drop down to show useful links

### ISSUE THREAD

**Issue by axelstudios** [SEED-platform/seed#2014]
Right now you can only create a new dataset, which is nearly useless
![image](https://user-images.githubusercontent.com/411466/68888172-aa1d3e80-06d7-11ea-8033-20fac0068e48.png)

Personally I would like to see links for the current organization (or sub-organization), like `Column Settings`, `Column Mappings`, `Data Quality`, `Cycles`, `Labels`, etc.  I find that's what I spend the most time navigating to get to

**Comment by RDmitchell** [comment]
@axelstudios -- I know that you all work with a lot of orgs or sub orgs open, and this would be useful to you. However, I don't think any of the cities that I work with that use SEED would ever need that functionality.

It might be useful for a group that had a lot of sub-orgs, such as a state program (like maybe the CEC) but I don't know of anyone that is currently using SEED that way (although I don't necessarily know all the users). 

But I suspect that no one uses the + to make a new data set either. So it's current implementation probably isn't very useful.

**Comment by axelstudios** [comment]
Yeah, if there are links that cities use more often then that would be a better replacement.  Mostly I just want that drop down to be more useful than it currently is
